src/staging/staging_main.py

```diff
@@ -151,6 +152,18 @@ def run_staging(
     check_file_exists(mapper_path)
     mapper = read_csv(mapper_path)
 
+    # Load cora mapper
+    StagingMainLogger.info("Loading Cora status mapper file")
+    cora_mapper_path = paths["cora_mapper_path"]
+    check_file_exists(cora_mapper_path)
+    cora_mapper = read_csv(cora_mapper_path)
+    #validates and updates from int64 to string type
+    cora_mapper = validate_cora_df(cora_mapper)
```

Comment:
I would swap these 2 validation steps so that the datatypes are forced first and then validated using your function